web-runtime/npm-package/src/__test__/test.spec.ts

```diff
@@ -1,75 +1,60 @@
-import { init } from '..';
 import fs from 'fs';
 import path from 'path';
-import { WASI } from '@wasmer/wasi';
-import { WasmFs } from '@wasmer/wasmfs';
-import bindings from '@wasmer/wasi/lib/bindings/browser';
+import { FaaS } from '../FaaS';
+import { callAvm } from '@fluencelabs/avm';
 
-const createModule = async (path: string) => {
-    const file = fs.readFileSync(path);
-    return await WebAssembly.compile(file);
-};
+const fsPromises = fs.promises;
 
-const invokeJson = (air: any, prevData: any, data: any, paramsToPass: any, callResultsToPass: any) => {
-    return JSON.stringify([
-        air,
-        Array.from(prevData),
-        Array.from(data),
-        paramsToPass,
-        Array.from(Buffer.from(JSON.stringify(callResultsToPass))),
-    ]);
-};
+const vmPeerId = '12D3KooWNzutuy8WHXDKFqFsATvCR6j9cj2FijYbnd47geRKaQZS';
 
 const b = (s: string) => {
     return Buffer.from(s);
 };
 
-const defaultAvmFileName = 'avm.wasm';
-const avmPackageName = '@fluencelabs/avm';
-
-const vmPeerId = '12D3KooWNzutuy8WHXDKFqFsATvCR6j9cj2FijYbnd47geRKaQZS';
-
-const _wasmFs = new WasmFs();
-
-const _wasi = new WASI({
-    // Arguments passed to the Wasm Module
-    // The first argument is usually the filepath to the executable WASI module
-    // we want to run.
-    args: [],
+const loadWasmModule = async (waspPath: string) => {
+    const fullPath = path.join(waspPath);
+    const buffer = await fsPromises.readFile(fullPath);
+    const module = await WebAssembly.compile(buffer);
+    return module;
+};
 
-    // Environment variables that are accesible to the WASI module
-    env: {},
+const examplesDir = path.join(__dirname, '../../../../examples');
 
-    // Bindings that are used by the WASI Instance (fs, path, etc...)
-    bindings: {
-        ...bindings,
-        fs: _wasmFs.fs,
-    },
-});
+describe('Fluence app service tests', () => {
```

Comment:
I still suggest to have tests for Redis and SQLite as marine-runtime do. If you think that it's too much for this PR, please add `TODO`.